Tests/SearchResultsTest.swift

```diff
@@ -581,4 +581,37 @@ class SearchResultsTest: XCTestCase {
             XCTFail("Failed to construct results")
         }
     }
+    
+    func testHierarchicalFacets() {
+        json["disjunctiveFacets"] = [
```

Comment:
Per documentation, is it not possible to do disjunctive faceting with hierarchical facets: https://www.algolia.com/doc/guides/searching/faceting/?language=swift#hierarchical-facets